style.css

```diff
@@ -79,23 +74,274 @@ p {
   height: 20px;
 }
 
-ul {
+.languages img {
+  padding-left: 20px;
+}
+
+.languages {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+}
+
+.languages > li {
+  display: flex;
+  border: 5px solid #f7f7f9;
+  border-radius: 10px;
+  margin: 3px;
+  background: #fff;
+}
+
+.zaf-social {
   display: inline-block;
   list-style: none;
-  margin-left: 10%;
 }
 
-li {
+.zaf-social li {
   display: inline-block;
   margin-right: 2%;
 }
 
-ul li a {
+.zaf-social li img {
   font-family: "Poppins", sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
   text-decoration: none;
+  margin-left: 240%;
+}
+
+.works {
+  display: grid;
+  width: 100%;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  background-color: #e5e5e5;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  margin: 5%;
+
+  /* border-right: 15%; */
+  max-width: 100%;
+  height: auto;
+
+  /* gap: 12px; */
+
+  /* border: 5px solid red; */
+  background: #fff;
+  border-radius: 16px;
+}
+
```

Comment:
- [OPTIONAL] You have a very good design for project cards, well done 🥇 I think it would be better if you add a little bit more `margin` between project cards, headline section and about section as present in the Figma design guidelines ✨ 
- [OPTIONAL] What do you think of using a `border` for project cards as present in the Figma design guidelines ? 💭 I know you can do that 💪
- [OPTIONAL] Please consider removing the commented code to make your codebase neat and clean😄

Your design 🔽 | Figma guidelines 🔽
---| ---|
![image](https://user-images.githubusercontent.com/106119105/229305465-16ce2f81-a2f1-46b9-b94a-1fd263f5d0c7.png) | ![image](https://user-images.githubusercontent.com/106119105/229305249-5e3a5f45-b748-43d3-bbc9-e8756a664059.png)

Reply:
@KanzaTahreem Done.